docker-compose.common.yaml

```diff
@@ -3,6 +3,7 @@
 services:
   jimm-base:
     environment:
+      JIMM_LOG_LOCAL: true
```

Comment:
Please assign it with a string.

Reply:
Could also just use same env var for level, but if "development" is sent, it'll use the dev config? So no need for another var?

Reply:
i think it's nice, but maybe too opinionated